### PR TITLE
fix name + description in update

### PR DESCRIPTION
**Pull request by dlasky**
fixes an issue where name and description were not properly initialized

## Testing

_Include any additional information about the testing you have completed to
ensure your changes behave as expected. For a speedy review, please check
any of the tasks you completed below during your testing._

- [ ] Added [unit tests](https://github.com/segmentio/action-destinations/blob/main/docs/testing.md#local-end-to-end-testing) for new functionality
- [ ] Tested end-to-end using the [local server](https://github.com/segmentio/action-destinations/blob/main/docs/testing.md#local-end-to-end-testing)
- [ ] [Segmenters] Tested in the staging environment

### CODE REVIEW

File: packages/cli-internal/src/commands/push.ts [hunk 1]
```diff
@@ -198,12 +198,14 @@ export default class Push extends Command {
         mobile: false
       }
 
+      const name = definition.name
```
Comment (dlasky) [inline packages/cli-internal/src/commands/push.ts:201]: this actually fixes the update portion below on Line 271 as name wasn't actually defined at that point in the code

**Review by NolanChan — APPROVED** [review]
After the fun hour journey, the fact that we never were able to update the name through `push` seems odd; this definitely needs to be a PSA of sorts. But LGTM overall

**Comment by dlasky** [comment]
agreed, i want to find out explicitly when that changed though first

**Review by LLuque-twilio — APPROVED** [review]
Approving so merging is available.

**Pull request merged**